tests/execution/default-value-input-type.md

```diff
@@ -0,0 +1,52 @@
+---
+skipped: true
+---
+
+# default value for input Type
+
+```graphql @config
+schema @upstream(baseURL: "http://abc.com") {
+  query: Query
+}
+
+type Query {
+  abc(input: Input!): Int @http(path: "/foo/{{.args.input.id}}")
+}
+
+input Input {
+  id: Int = 1
+}
+```
+
+```yml @mock
+- request:
+    method: GET
+    url: http://abc.com/foo/1
+  response:
+    status: 200
+    body: 1
+
+- request:
+    method: GET
+    url: http://abc.com/foo/2
+  response:
+    status: 200
+    body: 2
+```
+
+```yml @test
+- method: POST
+  url: http://localhost:8080/graphql
+  body:
+    query: >
+      query {
+        abc(input: {})
+      }
+- method: POST
+  url: http://localhost:8080/graphql
+  body:
+    query: >
+      query {
+        abc(input: {id:2})
+      }
+```
```

Comment:
Split this into two-tests. 
1. Where you don't make any requests. Mark this test as skipped.
2. Where we only test for merged and client schemas.

Reply:
done